src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -174,7 +174,12 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, MSBuild.TaskLog
                 }
                 else
                 {
-                    Log.LogError($"Item '{item}' already exists in {relativeBlobPath}.");
```

Comment:
This isn't the correct logic. if allowOverride == true and !blobExists then upload. Just don't do the exist check if allowOverride is set to true.

Reply:
```
bool blobExists = false;

if (!allowOverride) 
{
   blobExists = await feed.CheckIfBlobExists(relativeBlobPath);
}

if (allowOverwrite || !blobExists)
{
     Log.LogMessage($"Uploading {item} to {relativeBlobPath}.");
     UploadClient uploadClient = new UploadClient(Log);
     await uploadClient.UploadBlockBlobAsync(
         CancellationToken,
         feed.AccountName,
         feed.AccountKey,
         feed.ContainerName,
         item.ItemSpec,
         relativeBlobPath);
}
else
{
    Log.LogError($"Item '{item}' already exists in {relativeBlobPath}.");
}
```